### PR TITLE
feat: refine readme

### PR DESCRIPTION
**Pull request by HideBa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,14 @@
 # reearth-web
+This is front-end repository of [Re:Earth](https://github.com/reearth/reearth)
```
Comment (KaWaite) [inline README.md:2]:
```suggestion
This is the front-end repository of [Re:Earth](https://github.com/reearth/reearth)
```

File: README.md [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
 
-(unfinished)
+## Bug report and Feature requests
```
Comment (KaWaite) [inline README.md:4]:
```suggestion
## Bug reporting and requesting features
```

File: README.md [hunk 3]
```diff
@@ -1,7 +1,14 @@
...
+If your have any requests for new features all are welcome! Just create [an issue under Re:Earth repository](https://github.com/reearth/reearth/issues)
```
Comment (KaWaite) [inline README.md:5]:
```suggestion
Any requests for new features are very welcome! Just create [an issue under Re:Earth repository](https://github.com/reearth/reearth/issues) and the core team will take it into consideration.
```

File: README.md [hunk 4]
```diff
@@ -1,7 +1,14 @@
...
+When you found any bugs, please create bug issues from [bug report template](https://github.com/reearth/reearth/issues/new?assignees=&labels=&template=bug_report.md&title=).
```
Comment (KaWaite) [inline README.md:6]:
```suggestion
When you find a bug, please create a bug issue with the [bug report template](https://github.com/reearth/reearth/issues/new?assignees=&labels=&template=bug_report.md&title=).
```

File: README.md [hunk 5]
```diff
@@ -1,7 +1,14 @@
...
+
+
+## Discussions
+[Here](https://github.com/reearth/reearth/discussions) we have the discussion all topics.
```
Comment (KaWaite) [inline README.md:10]:
```suggestion
## Discussion
[Here](https://github.com/reearth/reearth/discussions) we have a dedicated area for discussions on all topics.
```

**Review by KaWaite — COMMENTED** [review]
Updated the English!

**Comment by HideBa** [comment]
@KaWaite Thanks a lot for your help!

**Pull request merged**